src/app/main-menu/main-menu.component.scss

```
@@ -0,0 +1,13 @@
+.example-form-field {
```

Klasy powinny nieść ze sobą jakieś znaczenie - być semantyczne.
```suggestion
.menu-form-field {
```